roles/ci_gen_kustomize_values/templates/include/ip-version.j2

```diff
@@ -0,0 +1,17 @@
+{%- if cifmw_ci_gen_kustomize_values_primary_ip_version == 4 -%}
+{%- set _ipv = {'ipvX': 'ipv4',
```

Comment:
might need a `namespace` in fact? seeing the issues raised in molecule

Reply:
Marking as resolved - something went sideways with the import / include approach.
I moved the map building to the vars instead.